src/base/utils/net.h

```diff
@@ -28,13 +28,23 @@
 
 #ifndef BASE_UTILS_NET_H
 #define BASE_UTILS_NET_H
+
 class QString;
 
 namespace Utils
 {
     namespace Net
     {
         bool isValidIP(const QString &ip);
+
+        template <typename T>
+        T unquote(const T &str)
```

Comment:
Sorry if this has caused you difficulties. But since it is used in several places, why not make it common? This does not interfere.
>As a string helper function yes, but no as a helper for http/net, only side effects.

Having side effects is bad idea (even such small and harmless at first glance). Do not forget that the calling code can change sometime, and it may cause big troubles (at least, we can't predict the consequences)!